app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User
+  attr_accessor :id
```

Comment:
What is the differentiation between id and name here?

Reply:
id represents a uid that gets included in the omniauth request. I think it's the id for the Google user.

name is the name of the Google user. For Example, that property for my account was "Andrew Wojciechowski"

I thought these properties might be useful for when we start limiting logins.

Reply:
I will add some comments explaining what these properties mean in more detail